pkg/subsystem/linux/rules.go

```diff
@@ -58,6 +58,14 @@ var (
 			"ubifs":    {"syz_mount_image$ubifs"},
 			"udf":      {"syz_mount_image$udf"},
 			"ufs":      {"syz_mount_image$ufs"},
+			"usb": {
```

Comment:
This will add usb subsystem for such bugs on top of any other subsystem we detected, right? Adding should be fine.

Reply:
It will give priority to `usb` if there are several subsystem candidates and all reproducers mention `usb`-related syscalls. In this case it will think usb was confirmed "twice" -- once by guilty files and once by reproducers.